src/QsCompiler/Tests.DocGenerator/DocParsingTests.cs

```diff
@@ -99,8 +99,8 @@ public void ParseUdt()
                 "```",
                 "",
                 "# See Also",
-                "- @\"microsoft.quantum.canon.paulievolutionset\"",
-                "- @\"microsoft.quantum.canon.evolutionset\""
+                "- Microsoft.Quantum.Canon.PauliEvolutionSet",
+                "- @\"Microsoft.Quantum.canon.evolutionset\""
```

Comment:
Due to details of our internal documentation generation pipeline, links formatted as `@""` still need to be all-lowercase:
```suggestion
                "- @\"microsoft.quantum.canon.evolutionset\""
```